packages/js/src/executors/verdaccio/verdaccio.impl.ts

```diff
@@ -84,22 +91,15 @@ function startVerdaccio(
             ? { VERDACCIO_STORAGE_PATH: options.storage }
             : {}),
         },
-        stdio: ['inherit', 'pipe', 'pipe', 'ipc'],
+        stdio: 'pipe',
```

Comment:
Now, this is equivalent to `stdio: 'inherit'`

Reply:
changed